docs/user-guide/configuring.md

```diff
@@ -63,7 +63,7 @@ Setting parser options helps ESLint determine what is a parsing error. All langu
 By default, ESLint uses [Espree](https://github.com/eslint/espree) as its parser. You can optionally specify that a different parser should be used in your configuration file so long as the parser meets the following requirements:
 
 1. It must be a Node module loadable from the config file where it appears. Usually, this means you should install the parser package separately using npm.
-1. It must conform to the [parser interface](/docs/developer-guide/working-with-plugins#working-with-custom-parsers).
+1. It must conform to the [parser interface](/docs/developer-guide/working-with-plugins#processors-in-plugins).
```

Comment:
Thanks for catching this! Agreed that the link is incorrect here. Could we actually link to `/docs/developer-guide/working-with-custom-parsers` instead?